taxi_service/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [os.path.join(BASE_DIR / "templates")],
```

Comment:
I mean `[BASE_DIR / "templates"]`

Like you have done with `STATICFILES_DIRS`
Do not forget to remove `os` import